### PR TITLE
Missing first_sync page restored + UI beautification

### PR DESCRIPTION
**Pull request by piotr-travisci**

### CODE REVIEW

File: app/services/auth.js [hunk 1]
```diff
@@ -197,8 +201,8 @@ export default Service.extend({
 
   syncingDidChange: observer('isSyncing', 'currentUser', function () {
     const user = this.currentUser;
-    if (user && user.get('isSyncing') && !user.get('syncedAt')) {
-      return this.router.transitionTo('first_sync');
+    if (user && !user.get('syncedAt')) {
+      this.first_sync = true;
```
Comment (sorta) [inline app/services/auth.js]: Hmm, I thought that `this.set` was still required for stuff like this, but I guess this seems to work?

Comment (piotr-travisci) [reply app/services/auth.js]: Yes it works perfectly. Thank you @sorta

**Comment by piotr-travisci** [comment]
> Hmm, I thought the tests were passing for this, but it looks like they're not?

Previous implementation worked in browser, but broke all tests regarding to broken tasks.
I reimplemented it and left "newbie" validation at API side.

File: app/models/owner.js [hunk 2]
```diff
@@ -18,8 +18,8 @@ export default VcsEntity.extend({
   name: attr('string'),
   login: attr('string'),
   channels: attr(),
-  isSyncing: attr('boolean'),
-  syncedAt: attr(),
+  isSyncing: attr(''),
+  syncedAt: attr(''),
```
Comment (sorta) [inline app/models/owner.js]: Does `attr('')` have a different effect than `attr()`? If not, then let's change these to `attr()` for consistency, it seems to be what we use more often

File: app/models/user.js [hunk 3]
```diff
@@ -16,8 +16,9 @@ export default Owner.extend({
   token: attr('string'),
   secureUserHash: attr('string'),
   gravatarId: attr('string'),
-  firstLoggedInAt: attr('date'),
+  firstLoggedInAt: attr(),
   allowMigration: attr('boolean'),
+  recentlySignedUp: attr(''),
```
Comment (sorta) [inline app/models/user.js]: Same here about `attr('')`

File: app/models/owner.js (hunk 2 above)
Comment (piotr-travisci) [reply app/models/owner.js]: @sorta Great finding. Thank you. I already corrected it.

File: app/models/user.js (hunk 3 above)
Comment (piotr-travisci) [reply app/models/user.js]: @sorta Great finding. Thank you. I already corrected it.

**Comment by piotr-travisci** [comment]
Thank you @sorta for quick review 🌷

**Pull request merged**